db/deploy/tables/rfi_data_status_type.sql

```diff
@@ -0,0 +1,28 @@
+-- Deploy ccbc:tables/rfi_data_status_type to pg
+
+begin;
+
+create table ccbc_public.rfi_data_status_type(
+  name varchar(1000) primary key,
+  description varchar(1000)
+);
+
+insert into ccbc_public.rfi_data_status_type values
+ ('draft', 'Draft'), ('sent', 'Sent');
+
+do
+$$
+begin
+
+perform ccbc_private.grant_permissions('select', 'rfi_data_status_type','ccbc_analyst');
```

Comment:
```suggestion
perform ccbc_private.grant_permissions('select', 'rfi_data_status_type','ccbc_analyst');
perform ccbc_private.grant_permissions('select', 'rfi_data_status_type','ccbc_admin');

```

Reply:
I was going to merge and update, it's working perfectly other than minor admin permissions. Though @AntBush needs to give the final approval since he requested changes :)

Reply:
Fixed